### ISSUE TITLE
iPad Keyboard Dismiss Annoyance

### ISSUE THREAD

**Issue by ash34** [isontheline/pro.webssh.net#1137]
Not sure if this intended behaviour or not on an iPad with the virtual keyboard. 

After pasting into the terminal the keyboard always dismisses immediately. It’s annoying as the very next thing the user will want to do is press the enter key or edit the pasted command. But instead the keyboard must be accessed again first.

Can the behaviour be changed or is there a reason for how it is now?

**Comment by isontheline** [comment]
Hello @ash34 👋

It's not a wanted behaviour, most probably a bug on my side.
But I would like a clarification please :

Are you using the "paste" action from the ellipsis to right icon or the "paste" action from the contextual menu (long press)?

The former should NOT hide the keyboard, the second could potentially hide it.

**Comment by ash34** [comment]
I am usually tapping the screen with 2 fingers.

**Comment by isontheline** [comment]
How do you use the 2 fingers combo on the iPad?
When I touch the screen the keyboard disappear and not contextual menu appear.

**Comment by ash34** [comment]
Yes like that. It pastes when I tap with 2 fingers. No other actions are needed.

**Comment by isontheline** [comment]
Perhaps you should use this option instead :
<img width="348" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/d9e21949-1d7e-43df-baa1-dfedc843ab4b">

It should remain the keyboard focused

**Comment by ash34** [comment]
Yes that solves it. Many thanks.

**Comment by isontheline** [comment]
You're Welcome @ash34
